vscode/src/ruby/chruby.ts

```diff
@@ -17,7 +17,7 @@ interface RubyVersion {
   version: string;
 }
 
-class RubyVersionCancellationError extends Error {}
+class RubyCancellationError extends Error {}
```

Comment:
```suggestion
class RubyInstallationCancellationError extends Error {}
```
(or similar, to make it a little more specific)